dune-project

```diff
@@ -34,7 +34,7 @@
  (depends
   (alcotest (and (>= 1.7.0) :with-test))
   (eio (= :version))
-  (mdx (and (>= 2.2.0) :with-test))
+  (mdx (and (>= 2.2.0) (< 2.4.0) :with-test))
```

Comment:
Any reason why <2.4.0 specifically? Is there a breaking change in 2.4.0?

Reply:
2.4 introduced OCaml 5.2 compatibility.

Reply:
2.4.0 started executing code while copying it. Looks like 2.4.1 has reverted this change (https://github.com/realworldocaml/mdx/releases/tag/2.4.1).

Reply:
Ah, got it. I guess in the future >2.4.0 <3.0.0 would also work, if they agree to stick to a semantic versioning logic.

Reply:
I've updated it to 2.4.1 in #712.